src/WebJobs.Script/Workers/WorkerConsoleLogService.cs

```diff
@@ -37,7 +41,12 @@ internal WorkerConsoleLogService(ILogger logger, IWorkerConsoleLogSource console
 
         public Task StartAsync(CancellationToken cancellationToken)
         {
-            _processingTask = ProcessLogs();
+            // Don't start processing logs if this is a standby configuration. This can lead
+            // to placeholder loggers capturing user logs during specialization.
+            if (!_options.IsStandbyConfiguration)
```

Comment:
yes -- @kshyju and i discussed this offline. He had a scenario where he used this for debugging locally but we came up with a better solution.

Reply:
Does this mean that anything that the placeholder used to log before, will either not be logged now or will be processed along with user logs? Any chance of placeholder logs getting mixed with user logs in kusto or reaching to appInsights?

Reply:
It's possible. And a good point. I'm going to lock this down a bit more so that we don't even emit logs to this path when in placeholder mode (they don't really go anywhere, anyway as there's no App Insights hooked up).

Reply:
> he had a scenario where he used this for debugging locally but we came up with a better solution.

Can you share more info about this and what the solution was? IIRC I also use logs for local debugging and would love to know if this is similar and would have an impact here